.pre-commit-config.yaml

```
@@ -25,7 +25,7 @@ repos:
     hooks:
       - id: cython-lint
   - repo: https://github.com/pre-commit/mirrors-clang-format
-    rev: v16.0.6
+    rev: v20.1.0
```

(Staying aligned with RAPIDS preferred clang-format version.)
```suggestion
    rev: v16.0.6
```